src/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/EmailAddressAttribute.cs

```diff
@@ -30,20 +30,28 @@ public override bool IsValid(object value)
 
             // only return true if there is only 1 '@' character
             // and it is neither the first nor the last character
-            bool found = false;
-            for (int i = 0; i < valueAsString.Length; i++)
+            var index = valueAsString.IndexOf('@');
```

Comment:
Nit: var => int

Reply:
Nit=?? You want me to change var for int?
```suggestion
            int index = valueAsString.IndexOf('@');
```

Reply:
Yes, please.